### PR TITLE
Added JWT claims registration requests to IANA

### PR DESCRIPTION
**Pull request by bc-pi**
Added JWT claims registration requests to IANA (to fix issue #315)

https://drafts.oauth.net/oauth-selective-disclosure-jwt/bc-IANA-JWT/draft-ietf-oauth-selective-disclosure-jwt.html#name-json-web-token-claims-regis

### CODE REVIEW

File: draft-ietf-oauth-selective-disclosure-jwt.md [hunk 1]
```diff
@@ -1114,7 +1114,31 @@ The work on this draft was started at OAuth Security Workshop 2022 in Trondheim,
 
 # IANA Considerations {#iana_considerations}
 
-TBD
+## JSON Web Token Claims Registration
+
+This specification requests registration of the following Claims in the
+IANA "JSON Web Token Claims" registry [@IANA.JWT] established by [@!RFC7519].
+
+Object Property Disclosure Digests:
```
Comment (Sakurann) [inline draft-ietf-oauth-selective-disclosure-jwt.md:1122]: do we need these headers `Object Property Disclosure Digests:`, `Array Element Disclosure Digest:`, `Disclosure Digest Hash Function:`..? they are the same as `claim description` values rn.

File: draft-ietf-oauth-selective-disclosure-jwt.md [hunk 2]
```diff
@@ -1114,7 +1114,31 @@ The work on this draft was started at OAuth Security Workshop 2022 in Trondheim,
...
+
+*  Claim Name: `_sd`
+*  Claim Description: Object property disclosure digests
```
Comment (Sakurann) [inline draft-ietf-oauth-selective-disclosure-jwt.md:1125]:
```suggestion
*  Claim Description: Disclosure digests
```
I am sorry, i could not understand `Object property disclosure digests`..? did you want to say that digests are hashes over objects?

File: draft-ietf-oauth-selective-disclosure-jwt.md [hunk 3]
```diff
@@ -1114,7 +1114,31 @@ The work on this draft was started at OAuth Security Workshop 2022 in Trondheim,
...
+*  Change Controller: IETF
+*  Specification Document(s):  [[ (#embedding_object_properties) of this specification ]]
+
+Array Element Disclosure Digest:
+
+*  Claim Name: `...`
+*  Claim Description: Array element disclosure digest
```
Comment (Sakurann) [inline draft-ietf-oauth-selective-disclosure-jwt.md:1132]:
```suggestion
*  Claim Description: Digests of Disclosures for array elements
```

**Review by Sakurann — CHANGES_REQUESTED** [review]
made suggestions to the claim descriptions as i was confused..

File: draft-ietf-oauth-selective-disclosure-jwt.md (hunk 1 above)
Comment (bc-pi) [reply draft-ietf-oauth-selective-disclosure-jwt.md:1122]: They are there to breakup the sets of bullets for each claim that run together without something. And I honestly don't know how else to do it with the markdown to xml to html/txt/etc tool chain in place. 

So we kinda need something but it doesn't have to be these exactly.

File: draft-ietf-oauth-selective-disclosure-jwt.md (hunk 2 above)
Comment (bc-pi) [reply draft-ietf-oauth-selective-disclosure-jwt.md:1125]: sure, if that's more understandable

File: draft-ietf-oauth-selective-disclosure-jwt.md (hunk 3 above)
Comment (bc-pi) [reply draft-ietf-oauth-selective-disclosure-jwt.md:1132]: ok but it would be singular

Comment (bc-pi) [reply draft-ietf-oauth-selective-disclosure-jwt.md:1132]: so how about this?
```suggestion
*  Claim Description: Digest of the Disclosure for an array element
```

File: draft-ietf-oauth-selective-disclosure-jwt.md (hunk 1 above)
Comment (Sakurann) [reply draft-ietf-oauth-selective-disclosure-jwt.md:1122]: I just saw this example https://www.rfc-editor.org/rfc/rfc7515#section-9.1.2, but yeah.
maybe use claim names as the `breakups`?

Comment (bc-pi) [reply draft-ietf-oauth-selective-disclosure-jwt.md:1122]: actually, maybe `<br/>` can be used to breakup with no text needed ....

Comment (bc-pi) [reply draft-ietf-oauth-selective-disclosure-jwt.md]: that seems to work ok

**Review by Sakurann — APPROVED** [review]

**Review by danielfett — APPROVED** [review]

**Pull request merged**